### DISCUSSION TITLE
[firebase_crashlytics] onError Listner

### DISCUSSION THREAD

**Discussion by guenth39** [firebase/flutterfire#6235] [Feature request]
I would like to have an onError listner for Crashlytics to write local logs or display central error messages. Could look something like this:

``` dart
  FirebaseCrashlytics.instance.onErrorListner = (dynamic exception, StackTrace? stack){
    //Do whatever you want
  };
```

**Comment by rrousselGit** [comment]
You could do something similar using `FlutterError.onError` or `runZoneGuarded` instead
